tests/test_flash_attn_ck.py

```diff
@@ -106,21 +136,22 @@ def test_flash_attn_qkvpacked(seqlen, d, dropout_p, causal, local, alibi, determ
     # of a Pytorch implementation.
     assert (out - out_ref).abs().max().item() <= 2 * (out_pt - out_ref).abs().max().item()
 
+    g = torch.randn_like(out)
+    if d <= 128 and d % 2 == 0:
```

Comment:
could we define something like `MAX_HEADDIM_SM8x`? then we can avoid hard-coded numbers

Reply:
d21811933213bb008e697293a049cba1fe9893e3